Cubical/Categories/CBPV/Instances/SmallStep.agda

```diff
@@ -0,0 +1,94 @@
+{-# OPTIONS --lossy-unification #-}
+
+module Cubical.Categories.CBPV.Instances.SmallStep where
```

Comment:
Why is this file separate from `CBPV.Instances.TransitionSystem`?

Reply:
I think transitions systems should live outside the CBPV subdirectory. 

I would think `CBPV.Instances.TransitionSystem` should contain a `CBPVModel` using transition systems and `CBPVHom` from a calculus with defined substitution to the `CBPVModel`.

Thoughts?

Reply:
The current file doesn't define an operational semantics though, it is just defining the CBPVModel of sets and transition systems. I think *this* file should be called `CBPV.Instances.TransitionSystem` and what is currently in `CBPV.Instances.TransitionSystem` should be moved to `Categories.Instances.TransitionSystem`